README.md

```diff
@@ -47,6 +47,34 @@ HTTP/1.1 200 OK
 I found 12 for $.value. But nothing for ${$.undefined}
 ```
 
+You can check every supported operators on the [Gatling JSONPath Syntax](https://github.com/gatling/jsonpath#syntax) documentation.  
+
+In addition to this syntax, you can specify a default value if nothing was found for a specific JSONPath.
```

Comment:
Maybe we could do only one example with all the "features".
Take the one above and make another sentence with "But I can put ${$.undefined§default} instead.